lnd.go

```diff
@@ -11,26 +11,22 @@ import (
 	"io/ioutil"
 	"net"
 	"net/http"
+	_ "net/http/pprof" // Blank import to set up profiling HTTP handlers.
```

Comment:
Can't keep the comment on a separate line, because auto-formatting will add a new line, which makes `impi` disapprove.